kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -571,34 +571,6 @@ def test_xcom_push(self):
         self.expected_pod['spec']['containers'].append(container)
         assert self.expected_pod == actual_pod
 
-    @mock.patch("airflow.providers.cncf.kubernetes.utils.pod_launcher.PodLauncher.start_pod")
-    @mock.patch("airflow.providers.cncf.kubernetes.utils.pod_launcher.PodLauncher.monitor_pod")
-    @mock.patch("airflow.kubernetes.kube_client.get_kube_client")
-    def test_envs_from_configmaps(self, mock_client, mock_monitor, mock_start):
```

Comment:
Moved it to `tests/providers/cncf/kubernetes/operators/test_kubernetes_pod.py` since this is a MOCK test just for KPO